odoo/service/server.py

```diff
@@ -1041,16 +1092,119 @@ def start(self):
         Worker.start(self)
         self.server = BaseWSGIServerNoBind(self.multi.app)
 
-class WorkerCron(Worker):
+
+class WorkerJobCommon(Worker):
+    def __init__(self, multi):
+        super(WorkerJobCommon, self).__init__(multi)
+        self.db_index = 0
+
+    def start(self):
+        os.nice(10)     # mommy always told me to be nice with others...
+        super(WorkerJobCommon, self).start()
+        if self.multi.socket:
+            self.multi.socket.close()
+
+    def _db_list(self):
+        if config['db_name']:
+            db_names = config['db_name'].split(',')
+        else:
+            db_names = odoo.service.db.list_dbs(True)
+        return db_names
```

Comment:
Why didn't this and the db_index stuff remain in WorkerCron? It seems unnecessary for and unused by WorkerAsync?

Reply:
`_db_list` is used by `process_work` and used for logging purprose